cloudhub-2/modules/ROOT/pages/ps-config-env.adoc

```diff
@@ -1,51 +1,82 @@
 For information, see xref:ch2-deploy-private-space.adoc[].
 
-You can associate a private space with the production and sandbox environments
-in the business group, and its child business groups, that it's registered with or with a subset of business groups.
+You can associate a private space with the production and sandbox environments in the business group, and its child business groups, that it's registered with or with a subset of business groups.
 
 By default, a private space is associated with all sandbox environments in its business groups.
 
-== Associate Environments and Business Groups
+== Associate Business Groups and Environments
 
 If you have the CloudHub Network Administrator permission, associate environments in the business group or groups with the private space:
 
 //SELECT PRIVATE SPACE SHARED
 include::partial$select-private-space.adoc[tag=selectPrivateSpace]
 include::partial$select-private-space.adoc[tag=clickPrivateSpaceName]
-. Click the *Environments* tab.
-. Select the environments from the drop-down list:
+. Click the *Environments* tab to:
+** <<associate-business-groups>>
+** <<associate-environments>>
+
+=== Associate Business Groups
+
+. Select the business groups from the drop-down list:  
 +
 --
-** *Any environment*
-+
-Associates the private space with all sandbox and production environments in the business group and its children.
-** *Sandbox environments*
+** *This business group and its children*
++ 
+This option allows you to select either all environments, all sandbox environment or all production environments for the selected business group and all its children.
+** *Selected business groups*
 +
-Associates the private space with only sandbox environments in the business group and its children.
-** *Production environments*
+Enter the business group name in the *Add business group* field and select it from the list. This option allows you to select specific environments for specific child business groups.
 +
-Associates the private space with only production environments in the business group and its children.
+The list includes the business group and its child business groups to which the private space is registered and that have not yet been added.
+
+[start=2]
+. Click *Save Changes* or *Discard Changes* to discard or confirm the association.
 --
-. Select the business groups from the drop-down list:
++
+image::ps-select-business-group.png[Select the business group from the drop-down list.]
+
+=== Associate Environments
+
+. Select the environments from the drop-down list:
 +
 --
-** *This business group and its children*
-** *Selected business groups*
+** *All Environment*
 +
-Enter the business group name in the *Add business group* field and select it from the list.
+Associates the private space with all sandbox and production environments in the business group and all its children, or in specific child business organizations.
```

Comment:
`specific child business organizations` -> same as above. You can select all environments for root org as well as it's children. So, can we change this to `specific business groups`. This should apply to all environment types.